src/tuples.py

```diff
@@ -17,28 +17,32 @@
 
 import math
 
+
 def dist(a, b):
     """Compute the distance between two x,y points."""
     x0, y0 = a  # Destructuring assignment
     x1, y1 = b
-    
+
     return math.sqrt((x1 - x0)**2 + (y1 - y0)**2)
 
+
 a = (2, 7)   # <-- x,y coordinates stored in tuples
 b = (-14, 72)
 
 # Prints "Distance is 66.94"
 print("Distance is: {:.2f}".format(dist(a, b)))
 
 
-
 # Write a function `print_tuple` that prints all the values in a tuple
 
-# YOUR CODE HERE
+def print_tuple(a):
+    for x in range(0, len(a)):
+        print(a[x])
+
 
 t = (1, 2, 5, 7, 99)
 print_tuple(t)  # Prints 1 2 5 7 99, one per line
 
 # Declare a tuple of 1 element then print it
-u = (1)  # What needs to be added to make this work?
+u = (1,)  # What needs to be added to make this work?
```

Comment:
Instead of adding the comma to make the argument a tuple (which is clever), you could look to see if the type of the argument is an int, and if it is, just print it. Then have an else which uses the for loop logic you have here.

Reply:
Thanks for the idea. I'll probably end up changing it in my next push